CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 
 #### Internal
 - Build universal binaries for release. XcodeGen now runs natively on Apple Silicon. [#1024](https://github.com/yonaskolb/XcodeGen/pull/1024) @thii
+- The `Core` target is renamed to avoid collisions with other packages. [#1057](https://github.com/yonaskolb/XcodeGen/pull/1057) @elliottwilliams
```

Comment:
Please add it to `Next Version`. This field is for already released version
https://github.com/yonaskolb/XcodeGen/pull/1057/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL7

Reply:
Will fix on master